packages/studio-web/src/app/upload/upload.component.html

```diff
@@ -134,15 +134,15 @@ <h2 i18n="Title for upload box" class="subtitle">
             i18n="Instructions for uploading audio"
             for="updateAudio"
             class="form-label"
-            >Use pre-recorded audio from an mp3, wav, or webm file.</label
+            >Use pre-recorded audio from an MP3 or WAV file.</label
```

Comment:
I'm not a fan of caps here I don't think

Reply:
Well, if you search online, the file extensions are lowercase, but the formats are generally upper case, which is why I changed it.